S23/akumar2709/main.py

```diff
@@ -0,0 +1,32 @@
+import csv
+
+#function converts word to lower case and provides special rules for irish and turkish since they are not covered by the book
+def toLower(input_string : str, language: str) -> str:
+  irish_vowels = "AEIOUÁÉÍÓÚ"
+  tilda_ord = 771
+#the following condition for irish uses a conditional lamda function which looks for the following rules 
+# - include a hyphen if the first alphabet is T or N and its followed by an irish vowel. It also checks for 
+# ord 771 since that means that there is a tild on the second alphabet since that could be achieved by either a singular unicode or by 2 unicodes
+# this is the reason we only check at postion[2] since if there is a tilda at [2] it means there can't be a vowel at [1]
+
+  if(language.find("ga") + 1):
+    irish_lower = lambda input_string : (input_string[0] + "-" + input_string[1:]).lower() \
```

Comment:
A bit odd to do this as a lambda.  Why not just apply these lines of code directly to the object with the same name?